downstream/modules/platform/proc-new-aap-instance-upgrade.adoc

```diff
@@ -0,0 +1,31 @@
+[id="proc-new-aap-instance-upgrade_{context}"]
+
+= Deploy a new instance for a side-by-side upgrade
+
+To proceed with the side-by-side upgrade process, deploy a second instance of Ansible Tower 3.8.x with the same instance groups configurations. This new instance will receive the content and configuration from your original instance, and will later be upgraded to {PlatformName} {PlatformVers}.
+
+== Deploy a new instance of Ansible Tower
+
+Start by downloading the Tower installer version that matches your original Tower instance version, then running the installer:
```

Comment:
Suggest rewording: To deploy a new Ansible Tower instance, download the Tower installer version that matches your original Tower instance version and run the installer:

Reply:
In this case, the user needs to download the older version (still called Tower) in order to receive the content from their original Tower instance, which will be upgraded to the newer versions (renamed Controller).